jsyntrax/src/test/java/org/atpfivt/jsyntrax/JSyntraxStyleFileTest.java

```diff
@@ -0,0 +1,155 @@
+package org.atpfivt.jsyntrax;
+
+import org.approvaltests.Approvals;
+import org.approvaltests.core.ApprovalFailureReporter;
+import org.approvaltests.core.Options;
+import org.atpfivt.jsyntrax.generators.SVGCanvas;
+import org.atpfivt.jsyntrax.generators.SVGCanvasBuilder;
+import org.atpfivt.jsyntrax.groovy_parser.Parser;
+import org.atpfivt.jsyntrax.styles.NodeStyle;
+import org.atpfivt.jsyntrax.styles.Style;
+import org.atpfivt.jsyntrax.styles.StyleConfig;
+import org.atpfivt.jsyntrax.units.Unit;
+import org.atpfivt.jsyntrax.units.tracks.stack.Stack;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.awt.*;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class JSyntraxStyleFileTest {
+
+    private final static Options options = new Options().forFile().withExtension(".svg");
+    private final SVGCanvasBuilder canvasBuilder;
+    private final Font testFont;
+
+    public JSyntraxStyleFileTest() {
+        testFont = getTestFont();
+
+        Style s = new Style(1, false);
+        s.title_font = testFont;
+
+        for (NodeStyle ns : s.nodeStyles) {
+            ns.font = testFont;
+        }
+        canvasBuilder = new SVGCanvasBuilder().withStyle(s);
+    }
+
+    private Font getTestFont() {
```

Comment:
This is a copy-paste from another test class. I think it's worth extracting into `TestUtils#getTestFont()` method. And by the way, we can extend `TestUtils` with things like `getTestFontSans`, `getTestFontSerif`, add font size as parameter etc.